tests/experimental.libsonnet

```diff
@@ -141,4 +101,52 @@ local volumes = import 'templates/volumes.libsonnet';
       },
     },
   },
+  // `BaseTpuVmMixin` is used to convert a 2VM target to 1VM.
+  BaseTpuVmMixin:: self.BaseTpuVmTest {
+    local config = self,
+    local cleanupHook = {
+      preStop: {
+        exec: {
+          command: [
+            'bash',
+            '/scripts/cleanup.sh',
+          ],
+        },
+      },
+    },
+
+    // Disable retries
+    jobTemplate+:: {
+      spec+: {
+        backoffLimit: 0,
+      },
+    },
+    podTemplate+:: {
+      spec+: {
+        containerMap+:: {
+          monitor: null,
+          train+: {
+            image: 'google/cloud-sdk',
+            lifecycle: cleanupHook,
+            envMap+:: {
+              LOCAL_OUTPUT_DIR: '/tmp/model_dir',
+              KUBE_GOOGLE_CLOUD_TPU_ENDPOINTS: if config.accelerator.replicas == 1 then
+                'local'
+              else
+                'tpu-$(POD_UID)',
+            },
+            resources+: {
```

Comment:
This should also be part of `BaseTpuVmTest`. This prevents tests from creating both 1VM and 2VM TPUs

Reply:
Done